meinberlin/assets/js/tabs.js

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
I kind of thought this would be the best bet as it is the definition of best practices but i'm not totally sure having been a big bootstrap user previously so adding own functionality for basic things like this is kind of new, maybe you know better @hom3mad3